src/components/Loader/LinearLoader.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import styled from 'styled-components';
+
+const Loader = () => {
+  return (
+    <LoaderContainer>
+      <LinearLoader />
+    </LoaderContainer>
+  );
+};
+
+export default Loader;
```

Comment:
nit: exported component objects should match the file name

maybe name the internal `styled(LinearProgress)` something different?